### PR TITLE
Improve documentation for Sysprep API

### PR DESCRIPTION
**Pull request by borod108**
Adding a note on a caveat when using the Sysprep tool.

Signed-off-by: borod108 <boris.od@gmail.com>

### CODE REVIEW

**Comment by dominikholler** [comment]
Thanks, looks good!

File: docs/virtual_machines/startup_scripts.md [hunk 1]
```diff
@@ -43,11 +43,13 @@ The general flow is:
     * [Sysprep Process Overview](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep-process-overview)
```
Comment (dominikholler) [inline docs/virtual_machines/startup_scripts.md:43]: Can you fix the line breaks here?

**Comment by dominikholler** [comment]
Are the information of the description in https://github.com/kubevirt/kubevirt/pull/4921 already in the documentation? If not, it might worth a new PR.

File: docs/virtual_machines/startup_scripts.md (hunk 1 above)
Comment (borod108) [reply docs/virtual_machines/startup_scripts.md:44]: @dominikholler done, thank you!

File: docs/virtual_machines/startup_scripts.md [hunk 2]
```diff
@@ -40,14 +40,17 @@ The general flow is:
...
 
+    Note: It is important that there is no answer file detected when the Sysprep Tool is triggered, because Windows Setup searches for answer files at the beginning of each configuration pass and caches it. If that happens, when the OS will start - it will just use the cached answer file, ignoring the one we provide through the Sysprep API. More information can be found [here](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/windows-setup-automation-overview#implicit-answer-file-search-order)
+
 2. Providing an Answer file named `autounattend.xml` in an attached media. The answer file can be provided in a ConfigMap or a Secret with the key `autounattend.xml`
 
     More information can be found here: [Answer files (unattend.xml)](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/update-windows-settings-and-scripts-create-your-own-answer-file-sxs)
 
-    Note that there are also many easy to find online tools available for creating an answer file.
+    Note that there are also many, easy to find, online tools available for creating an answer file.
```
[the remark this answers is not in the document]
Comment (borod108) [reply docs/virtual_machines/startup_scripts.md]: I do not even know why I touched this sentence, it is not part of this PR. Removed this change.

**Comment by borod108** [comment]
@mazzystr is it ok now?

File: docs/virtual_machines/startup_scripts.md [hunk 3]
```diff
@@ -40,9 +40,12 @@ The general flow is:
         %WINDIR%\system32\sysprep\sysprep.exe /generalize /shutdown /oobe /mode:vm
 
     More information can be found here:
+
     * [Sysprep Process Overview](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep-process-overview)
     * [Sysprep (Generalize) a Windows installation](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation)
 
+    Note: It is important that there is no answer file detected when the Sysprep Tool is triggered, because Windows Setup searches for answer files at the beginning of each configuration pass and caches it. If that happens, when the OS will start - it will just use the cached answer file, ignoring the one we provide through the Sysprep API. More information can be found [here](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/windows-setup-automation-overview#implicit-answer-file-search-order)
```
Comment (mazzystr) [inline docs/virtual_machines/startup_scripts.md:47]: `Note:` Let's get this changed to a real annotation.  See this [page](https://github.com/kubevirt/user-guide/blame/master/docs/web_console.md#L3)

After this update we are ready.  Thanks!

**Review by mazzystr — CHANGES_REQUESTED** [review]

File: docs/virtual_machines/startup_scripts.md (hunk 3 above)
Comment (mazzystr) [reply docs/virtual_machines/startup_scripts.md:47]: See red arrow in img .... 
img

The section is very busy.  We need some line breaks.  I think this note should be encapsulated into an annotation like the proposed L53.  Use `!!! Note` to engage the annotation feature.

Comment (borod108) [reply docs/virtual_machines/startup_scripts.md]: @mazzystr is it better now?

Comment (mazzystr) [reply docs/virtual_machines/startup_scripts.md]: Looks great.  Thanks.

**Comment by mazzystr** [comment]
You got it!  Thanks for the contribution!

/lgtm
/approve

**Review by mazzystr — APPROVED** [review]

**Pull request merged**